src/gui/programupdater.cpp

```diff
@@ -127,18 +127,18 @@ void ProgramUpdater::rssDownloadFinished(const Net::DownloadResult &result)
 
         if (xml.isStartElement())
         {
-            if (xml.name() == "item")
+            if (xml.name() == QLatin1String("item"))
                 inItem = true;
-            else if (inItem && xml.name() == "link")
+            else if (inItem && xml.name() == QLatin1String("link"))
```

Comment:
It would be best if you also add parentheses: `(xml.name() == QLatin1String("link"))`